diff --git a/node_modules/@wix/services-definitions/build/core-services/signals/definition.d.ts b/node_modules/@wix/services-definitions/build/core-services/signals/definition.d.ts
index 3735ac3..57d5eed 100644
--- a/node_modules/@wix/services-definitions/build/core-services/signals/definition.d.ts
+++ b/node_modules/@wix/services-definitions/build/core-services/signals/definition.d.ts
@@ -1,5 +1,6 @@
 export type ReadOnlySignal<T> = {
     get: () => T;
+    peek: () => T;
 };
 export type Signal<T> = ReadOnlySignal<T> & {
     set: (newValue: T) => void;
